### PR TITLE
Removes GMM from Moonstation

### PR DESCRIPTION
**Pull request by YakumoChen**
## About The Pull Request
As title

## Why It's Good For The Game

Making Speedbike money four times over in a shift was only funny once

## Proof Of Testing

Just a map change. Didn't compile on my end but the build issue was somewhere unrelated so *shrug*

## Changelog

:cl:
del: Nanotrasen has removed the Galactic Minerals Market from their Moonstation branch due to repeated market crashing by crew.
/:cl:

### CODE REVIEW

File: _maps/map_files/moonstation/moonstation.dmm [hunk 1]
```diff
@@ -64627,14 +64621,14 @@
 /area/station/security/brig/entrance)
 "rTb" = (
 /obj/effect/turf_decal/siding/dark,
-/obj/machinery/materials_market,
```
Comment (YakumoChen) [inline _maps/map_files/moonstation/moonstation.dmm:64630]: see here for relevant code change since mapmerge likes to re-organise everything

**Comment by NargaInDisguise** [comment]
Merge it, it keeps happening.

**Review by Majkl-J — APPROVED** [review]

**Comment by Majkl-J** [comment]
Why is the map diff so massive you removed a chair and machine

**Comment by YakumoChen** [comment]
> Why is the map diff so massive you removed a chair and machine

Map merge plus using byond's native editing tools

**Pull request merged**